.vscode/c_cpp_properties.json

```diff
@@ -1,43 +1,79 @@
 {
     "env": {
-        "BOLOS_SDK": "/opt/nanos-secure-sdk",
-        "BOLOS_ENV": "/opt/bolos-devenv"
+        // Change to point to local copy of the two ledger sdks
+        "LEDGER_SDK": "~/workspaces/ledger-secure-sdk",
```

Comment:
already defined in `settings.json ` ?

Reply:
Indeed but unfortunately not accessible from c_cpp_properties.json (as far as I saw)

Reply:
Did an fix to only have to define local sdks in c_cpp_properties.json